canonical-web-and-design/ubuntu.com
# Display details for a UA subscription

## Done

- Added utilities for displaying subscription data.
- Display details for a UA subscription in the details panel.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10291.demos.haus/advantage?test_backend=true).
- Click on a few of the top cards in the subscription list (they all say "Free Personal Token" until https://github.com/canonical-web-and-design/ubuntu.com/pull/10290 lands).
- The title and top columns (created, machine type etc.) should display the data from the API.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/109.

## Screenshots

<img width="632" alt="Screen Shot 2021-09-03 at 1 15 49 pm" src="https://user-images.githubusercontent.com/361637/131945440-65d41aa9-081d-4b31-a30e-caefbd1e8723.png">
